spec/acceptance/agent_spec.rb

```diff
@@ -77,6 +85,9 @@ class { 'zabbix::agent':
               zabbix_version       => '#{version}',
             }
           EOS
+
+          prepare_host
```

Comment:
same here

Reply:
Yes, but I'm work on the other indempotent test to use shared example, if it was merged I can rebase this code